pkg/cmd/grafana-cli/commands/reset_password_command.go

```diff
@@ -2,18 +2,26 @@ package commands
 
 import (
 	"fmt"
+	"syscall"
 
 	"github.com/fatih/color"
 	"github.com/grafana/grafana/pkg/bus"
 	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/util"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
you need to add this to the vendor folder with dep.

`dep ensure -add golang.org/x/crypto/ssh/terminal`